addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -305,8 +305,8 @@ def _l10n_es_edi_get_invoices_info(self, invoices):
 
                 mod_303_10 = self.env.ref('l10n_es.mod_303_casilla_10_balance')._get_matching_tags()
                 mod_303_11 = self.env.ref('l10n_es.mod_303_casilla_11_balance')._get_matching_tags()
-                tax_tags = invoice.line_ids.tax_tag_ids
-                intracom = any(tax in tax_tags for tax in mod_303_10) and any(tax in tax_tags for tax in mod_303_11)
+                tax_tags = invoice.line_ids.tax_ids.repartition_line_ids.tag_ids
```

Comment:
@oco-odoo Here, it is just detecting the operation as intracom or not.  And weirdly enough, for regular operations, they go into an intra-community line in the report, but not for the credit notes (they go into a generic line)  So, it is not about tax configuration being changed or something.  

In that respect, 303 and SII will be different as in 303 you won't know if the credit note was intracom.

Reply:
But you're getting the tags on the repartition line, not on the aml, so you depend on the configuration of the taxes :thinking: 
Why not just invoice.line_ids.tax_tag_ids ?

Reply:
discussed: we have to pass through the tax, because we also want to include the refund, but only the invoice repartition contains a tag that allows identifying the tax without error. It hence is possible to have a difference due to a change in the configuration of the tax, though it does not make sense functionally and we'll live with it.